### ISSUE TITLE
Broken Translate link in About

### ISSUE THREAD

**Issue by kudlav** [schwabe/ics-openvpn#1703]
# General information
1. Android 14
2. Android Vendor ROM
3. Device Pixel 7a
4. Version of the app 0.7.51 F-droid
 
# Description of the issue

Visit About screen. The following link is broken: 

*You can help translating by visiting https://crowdin.net/project/ics-openvpn/invite*

> Something went wrong
>
>Project does not exist or access to this project is limited.
>
>In case you are confident that you have access to the project please login.

**Comment by PeterDaveHello** [comment]
Same here, @schwabe any ideas? Thanks.

**Comment by PeterDaveHello** [comment]
Weblate also Page Not Found: https://hosted.weblate.org/projects/openvpn-for-android/

**Comment by schwabe** [comment]
Use https://crowdin.com/project/ics-openvpn

**Comment by schwabe** [comment]
@PeterDaveHello https://hosted.weblate.org/ is a different service.

**Comment by PeterDaveHello** [comment]
> @PeterDaveHello [hosted.weblate.org](https://hosted.weblate.org/) is a different service.

yes, I mentioned it because of seeing https://github.com/schwabe/ics-openvpn/pull/1455 :sweat_smile: That's a great translation platform for open source project :smile:

**Comment by schwabe** [comment]
Yeah, there was a previous attempt to put this to weblate but there were some problems with the license iirc and there was no proper way to switch to weblate without loosing the existing translations, so basically that did not lead anywhere as I also didn't have the time at that time to spend to solve these issues, especially importing the older translations. Asking all translators to retranslate again is not something that I want to do.

**Comment by PeterDaveHello** [comment]
Hey @schwabe,

Do you think we could give Weblate another shot for translations? It seems pretty good for open-source projects. I noticed a lot of free software uses it. Just wondering if ICS-OpenVPN's license doesn't fit their requirements?

Thanks!

**Comment by schwabe** [comment]
Well converting existing translations is something that needs to be addressed. The last guy that wanted to convert did a lot of things because he assumed that were better and overall impression was that it has to be his way (see for example that PR https://github.com/schwabe/ics-openvpn/pull/1477) and put a lot of pressure on me to do this when I was busy with other things. 

The problem with the license is that this project is GPL2+ + exception to allow linking against Apache and that license was not something that could be setup in the free weblate OSS hosting and I didn't want to lie about my license either. This is something that could be probably easily been solved with the support of weblate but when someelse creates a weblate project for that has a 14 day limit to address all the outstanding issues when you don't really have time for that, isn't that helpful, so ultimately, I failed to fulfill the requirements to be an OSS accepted project and the project got deleted again.

**Comment by PeterDaveHello** [comment]
@schwabe Thanks for taking the time to share your thoughts. The most valuable part of open source is the care and effort every contributor puts in — protecting that is what truly matters. There's no need to rush improvements; better solutions will come when the time is right. I really appreciate your thoughtful approach and everything you've put into this.
